### PR TITLE
Merge library to development

### PR DESCRIPTION
**Pull request by Blasco9**
In this branch, we:
- Created the HTML index
- Created the styles for the page
- Designed the logic responsible for creating, removing and updating books
- Updated the README file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,57 +1,33 @@
...
 
-- Major languages,
-- frameworks,
-- technologies used
+- Javascript,
+- CSS
 
 ## Live Demo
 
 [Live Demo Link](https://livedemo.com)
```
Comment (Torres-ssf) [inline README.md]: - [x] The live demo link is broken :cry: . We need to fix that :smile:

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes required :muscle: 
Hi team :hand: 
Awesome job so far :clap: 
But we need to work on some improvements before merging this PR.

- [x] Right it is possible to create a book without filling any information in the form. We need to fix that and make it possible to create books only after giving the required information: `title`, `author`, `page`.

- [x] It should not be possible to create books with a negative number of pages, like `-150`

- [x] We can improve the user experience from your application by adding headers to each book, to inform what is the `title`, what is the `author` and the `pages`. I agree that the current approach it's pretty intuitive, but some users may not think the same way.

Please kindly submit for code review again after making these changes.
Feel free to message me on slack if you have any questions :smile: 
Happy Coding :computer: 
**Sergio Torres**

File: index.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+		<div class="container"></div>
+
+		<div class="modal">
+			<form class="book-form">
+        <label for="title">Title</label>
+        <input class="book-title" name="title" type="text" />
+        <label for="author">Author</label>
+        <input class="book-author" name="author" type="text" />
+        <label for="pages">Pages</label>
+        <input class="book-pages" name="pages" type="number" />
```
Comment (shshamim63) [inline index.html:19]: [Optional]
Usually, a book does not have any negative page numbers. So allowing a user to do that is against the default convention. So you can control this scenario, by updating the input field in the following way,
```html
<input class="book-pages" name="pages" type="number" min="0"  />
```

**Review by shshamim63 — APPROVED** [review]
## :100: Approved
___
Hi @Blasco9 and @Ashah15 :wave: 
Congratulation, you have successfully completed this project :clap: , nothing is blocking me from approving this project. You can go ahead.
___
## :herb: Happy Coding :herb: 
### Shakhawat Hossain

**Pull request merged**